docs/FRSKY_R9_TX.md

```diff
@@ -0,0 +1,171 @@
+
+#### ESP32 Wireless Bridge | Programming the M5Stamp C3U Mate ####
+
+__Be sure to unplug the M5Stamp C3U Mate from the back of the R9M when programming via USB to avoid feeding 5 volt power back to R9M which might cause damage.__
+
+To install the sketch on the M5Stamp C3U Mate:
+  1. Open the Arduino IDE.
+  2. If not already installed, install the ESP32 core using the Boards Manager.
+  3. Open the mlrs-wireless-bridge.ino sketch from the mLRS esp/mlrs-wireless-bridge folder.
+  4. Edit the sketch to uncomment only the MODULE\_M5STAMP\_C3U\_MATE\_FOR\_FRSKY\_R9M define.
+  5. Select the ESP32C3 Dev board in the IDE.
+  6. Connect the M5Stamp C3U Mate via USB to your computer while holding down the center button.
+  7. Upload the sketch.
+
+#### ESP32 Wireless Bridge | Programming the M5Stamp Pico Mate ####
+
+Programming the Pico Mate is straightforward using a USB TTL serial adapter which supports the DTR and RTS pins. The adapter which comes with the [M5Stamp Pico DIY Kit](https://shop.m5stack.com/products/m5stamp-pico-diy-kit) is the most convenient and the kit includes the Pico Mate module. It is recommended to solder the 6 pin female header connector to the appropriate pins on the module and plug in the kit's serial adapter directly. Alternatively, if you prefer to leave off the programming connector, it is possible to insert the serial adapter pins into the appropriate thru-holes on the module and hold it in place with some pressure at an angle as to ensure continuous contact with all 6 pins during the programming process.
+
+To program the M5Stamp Pico Mate follow the steps above for the M5Stamp C3U Mate adjusting to the M5Stamp Pico Mate where appropiate. You can select the Bluetooth Classic protocol if you prefer.
+
+#### ESP32 Wireless Bridge Hardware Installation ####
+
+Connecting the M5Stamp C3U Mate or Pico Mate to the R9M is easy:
+  1. Remove the M5Stamp module's screw and plastic cover. 
+  2. Cut a 5 pin length of the included pin header female connector and pull out the pin from the last position. 
+    - This position serves as a key to avoid plugging in the board incorrectly. 
+  3. Solder the pins in the thru holes as shown below (key position hanging over the left) and reinstall the plastic cover.
+  4. Program the sketch onto the board, detailed below.
+  5. Install on the back of the R9M as shown above. 
+  6. You can then connect your GCS computer or mobile device to the "mLRS AP UDP" WiFi access point and connect the GCS via UDP on port 14550.
+
+<br> <img src="images/Frsky_R9M_M5Stamp_headers.png" width="720px">
+
+### Cooling ###
```

Comment:
should be ## xx ## titled, as now is sorted

Reply:
Updated